test/JDBC/input/BABEL-3550.mix

```diff
@@ -0,0 +1,32 @@
+-- tsql
+USE master
+GO
+CREATE LOGIN babel_3550_login1 WITH PASSWORD='12345678';
+GO
+
+ALTER SERVER ROLE sysadmin ADD MEMBER babel_3550_login1;
+GO
+
+-- tsql  user=babel_3550_login1  password='12345678'
```

Comment:
I think the correct format for password here should be without the quote mark. 
For some reason JDBC framework won't detect the error in password in local, but will report error when ran on prod instances.